src/message/index.ts

```diff
@@ -41,9 +41,9 @@ export interface Headers extends IncomingHttpHeaders {
  * @property {@linkcode Headers} `headers` - the headers for the event Message
  * @property string `body` - the body of the event Message
  */
-export interface Message {
+export interface Message<T> {
```

Comment:
I believe all generic types require a default value to not be a breaking changes.

```ts
export interface Message<T = string> {
    headers: object;
    body: T | Buffer | unknown;
}
```

If this is a breaking change, please change the conventional commit type to `feat!`.